OPHD/Things/Structures/FusionReactor.h

```diff
@@ -26,6 +47,12 @@ class FusionReactor : public Structure
 
 	void defineResourceOutput() override
 	{
-		resourcesOut().energy(1000);
+		resourcesOut().energy(calculateEnergyProduced());
+	}
+
+private:
+	int calculateEnergyProduced()
+	{
+		return operational() ? FUSION_REACTOR_BASE_PRODUCUCTION : 0;
```

Comment:
operational check occurs when all energy sources are aggregated. So this call isn't necessary for defineResourceOutput. I thought it would be best to try to standardize the call within FusionReactor though. 

I suspect in the future energy production will become more complex, esp with research tree.

Reply:
I had noticed that. Looks like it helps for display purposes though.